mcs/class/corlib/System/MulticastDelegate.cs

```diff
@@ -112,167 +109,141 @@ public sealed override int GetHashCode ()
 		// </summary>
 		public sealed override Delegate[] GetInvocationList ()
 		{
-			MulticastDelegate d;
-			d = (MulticastDelegate) this.Clone ();
-			for (d.kpm_next = null; d.prev != null; d = d.prev)
-				d.prev.kpm_next = d;
-
-			if (d.kpm_next == null) {
-				MulticastDelegate other = (MulticastDelegate) d.Clone ();
-				other.prev = null;
-				other.kpm_next = null;				
-				return new Delegate [1] { other };
-			}
-
-			var list = new List<Delegate> ();
-			for (; d != null; d = d.kpm_next) {
-				MulticastDelegate other = (MulticastDelegate) d.Clone ();
-				other.prev = null;
-				other.kpm_next = null;
-				list.Add (other);
-			}
-
-			return list.ToArray ();
+			return delegates;
 		}
 
 		// <summary>
-		//   Combines this MulticastDelegate with the (Multicast)Delegate `follow'.
+		//   Combines this MulticastDelegate with the (Multicast)Delegate `other'.
 		//   This does _not_ combine with Delegates. ECMA states the whole delegate
 		//   thing should have better been a simple System.Delegate class.
 		//   Compiler generated delegates are always MulticastDelegates.
 		// </summary>
-		protected sealed override Delegate CombineImpl (Delegate follow)
+		protected override sealed Delegate CombineImpl (Delegate v)
```

Comment:
What's the name of the argument on reference sources? 
Arg names must match what RS has as it's needed for names argument calls to work.